src/main/java/com/stripe/model/EventDataObjectDeserializer.java

```diff
@@ -75,12 +74,14 @@
   }
 
   /**
-   * Gets data event object, in favor of the deprecated {@link EventData#getObject()}.
-   * When non-, the deserialized {@code StripeObject} preserves high data integrity because of
-   * correspondence between schema of the API response and the model class (the underlying
-   * concrete class for abstract {@code StripeObject}) schema. This is when
-   * {@link Event#getApiVersion()} matches {@link Stripe#API_VERSION}.
-   * @return stripe object that fully represent its original raw JSON response.
+   * Gets an option of data event object, in favor of the deprecated {@link EventData#getObject()}.
```

Comment:
Nice Mick! Looks mostly fine, but I'd probably also go with what OB suggested and make "option" a link or code:

> ==. Maybe {@link java.util.Optional} would work? Otherwise {@code Optional<T>} is probably explicit enough too.